test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/054_ATF_P_TC_media_PTU_brings_certificate_RPC_ForceProtectedService_ON.lua

```diff
@@ -0,0 +1,173 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate has NO "certificate" and "ForceProtectedService"=OFF at .ini file
```

Comment:
@dboltovskyi, thank you for notice, done in 517716d6375db8ae2977579ec98c0487470c0dae